### PR TITLE
add <minhaj-313>

### LINKED ISSUE #1766: .

### PR DESCRIPTION
**Pull request by minhaj-313**
resolves #1766 

## Checklist for profile addition

- [x] Name the PR `fix: add <your-username>`.
- [x] You've added file named `your-username.json` in the [profiles](https://github.com/EddieHubCommunity/awesome-github-profiles/tree/main/profiles) directory.
- [x] You've linked the PR to the correct issue.

---

## Other additions/changes

- [x] Changes made in this PR are mentioned below
- [x] Search previous suggestions before making a new one, as yours may be a duplicate.
- [x] I acknowledge that all my contributions will be made under the project's license.
- [x] I've read the [contributing guidelines](https://github.com/EddieHubCommunity/awesome-github-profiles/blob/main/CONTRIBUTING.md)

### CODE REVIEW

**Review by PiseAtharva — COMMENTED** [review]
Hey Minhaj, really appreciated the profile, but I think you missed the check marks in your PR (it should be similar to #1759 just for your reference )

**Review by schmelto — APPROVED** [review]

**Comment by minhaj-313** [comment]
> Hey Minhaj, really appreciated the profile, but I think you missed the check marks in your PR (it should be similar to #1759 just for your reference )

I Updated now accept my PR

**Comment by PiseAtharva** [comment]
> I Updated now accept my PR

Tom already did that for ya 😅

**Comment by minhaj-313** [comment]
> > I Updated now accept my PR
> 
> Tom already did that for ya 😅

You guys haven't done it yet, Merge it.

File: profiles/minhaj-313.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "image": "https://user-images.githubusercontent.com/107562768/226933817-9212dc22-f422-44d8-8aa2-888bb6d43878.png",
+  "issueId": 1766,
+  "name": "Shaikh Minhaj",
+  "username": "minhaj-313"
+}
```
Comment (Panquesito7) [inline profiles/minhaj-313.json:6]: Missing an endline. Without this endline, there could be issues with some IDEs and configurations.
```suggestion
}

```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Review by getlost01 — APPROVED** [review]
Fine

**Review by minhaj-313 — COMMENTED** [review]
Please Merge my PR

**Review by Pradumnasaraf — CHANGES_REQUESTED** [review]
Hey,

Revert changes from another file. We don't allow modification is someone's profile.

**Pull request closed without merging**